tests/unit/date-mixin.spec.ts

```diff
@@ -35,6 +35,12 @@ describe('Date Mixin', () => {
     expect(vm.yyyyMmDdToDate('2021-07-01').toISOString()).toEqual('2021-07-01T07:00:00.000Z') // PDT
   })
 
+  it('returns correct values for mmmDdYyyyToDate()', () => {
+    expect(vm.mmmDdYyyyToDate(null)).toBeNull()
+    expect(vm.mmmDdYyyyToDate('January 1, 2021').toISOString()).toEqual('2021-01-01T08:00:00.000Z') // PST
+    expect(vm.mmmDdYyyyToDate('July 1, 2021').toISOString()).toEqual('2021-07-01T07:00:00.000Z') // PDT
+  })
+
```

Comment:
Tests for new Date Mixin method 🎉

Reply:
Probably don't need this (I think because you got rid of most of the new Date()), but maybe we could write some tests with:
https://github.com/Jimbly/timezone-mock

Reply:
I got rid of all `new Date` with no formal parameters and with improper date strings, but I definitely still use `new Date` to create date objects.

I tried the timezone-mock package (great idea btw!) but unforunately it doesn't seem to support `new Date` with a timezone options parameter, ie:

![image](https://user-images.githubusercontent.com/10002889/188917354-53cdbea3-bab2-4c1e-8fe2-4d06a21a9bee.png)

See [this commit](https://github.com/bcgov/business-edit-ui/pull/371/commits/99159807515e96ab692920ea1ee0e3dd8f632a98) for details. I will now push another commit to revert these changes.

Reply:
^^ actually, it's still unhappy after I remove the options parameter. I guess it doesn't like the date format from `toLocateString`.

I'll play around a bit with this but I might not be able to use it.

Reply:
Ahh that's too bad, worth a shot

Reply:
OK, I've spent enough time trying to get this to work (unsuccessfully)...

Looking at the library's source code, it seems that it can't parse the date-time string created by toLocaleString. Specifically, the library only knows about certain ISO-ish formats, which I can't seem to create reliably using toLocaleString.